examples/ENS/ENS.xml

```diff
@@ -35,15 +46,6 @@
   <!-- emitted via: function registerWithConfig(string memory name, address owner, uint duration, bytes32 secret, address resolver, address addr) public payable -->
   <ts:contract name="ETHRegistrarController">
     <ts:address network="1">0x283af0b28c62c092c9727f1ee09c02ca627eb7f5</ts:address>
```

Comment:
@colourful-land how does the `NameRegistered` event get "bound" to the contract `0x283af0b28c62c092c9727f1ee09c02ca627eb7f5`?